src/constants.ts

```diff
@@ -39,4 +39,12 @@ export class ContextTestingHelper {
 	getPath(key: string) {
 		return this._context.store[key];
 	}
+
+	createSpecFileAtWorkingDir(){
+		fs.writeFileSync(path.resolve(process.cwd(), 'asyncapi.yml'), '');
```

Comment:
You should also check if spec file exists in the working dir by `fs.existsSync(path.resolve(process.cwd(), 'asyncapi.yml'))` statement.